ReactiveCocoa/NSObject+Association.swift

```diff
@@ -1,5 +1,29 @@
 import ReactiveSwift
 
+internal struct AssociationKey {
```

Comment:
I don't think that optimization is worth it in that case. Unless there's big benefit to performance, we should optimize of clarity.